modules/manage/pages/iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc

```diff
@@ -0,0 +1,203 @@
+. In *Configure Service Connection*, provide a name. Open Catalog will also create a new principal with this name.
+. Make sure *Create new principal role* is toggled on.
+. Enter a name for the principal role. Then, click *Create*.
+
+After you create the connection, you'll be provided the client ID and client secret. Save these credentials to add to your cluster configuration in a later step.
+
+=== Create a catalog role
+
+Grant privileges to the principal created in the previous step:
+
+. In Open Catalog, select *Catalogs*, and select your catalog.
+. On the *Roles* tab of your catalog, click *+ Catalog Role*.
+. Give the catalog role a name. 
+. Under *Privileges*, select `CATALOG_MANAGE_CONTENT`. This provides full management https://other-docs.snowflake.com/en/opencatalog/access-control#catalog-privileges[privileges] for the catalog. Then, click *Create*. 
+. On the *Roles* tab of the catalog, click *Grant to Principal Role*. 
+. Select the catalog role you just created.
+. Select the principal role you created earlier. Click *Grant*.
+
+=== Update cluster configuration
+
+In this step, you'll configure your Redpanda cluster to enable Iceberg on a topic, as well as set up the integration with Open Catalog.
+
+. Edit your cluster configuration to set the `iceberg_enabled` property to `true`, and set the catalog integration properties listed in the example below. You must restart your cluster if you change this configuration for a running cluster. You can run `rpk cluster config edit` to update these properties:
++
+[,bash]
+----
+iceberg_enabled: true 
+iceberg_catalog_type: rest
+iceberg_rest_catalog_endpoint: https://<snowflake-orgname>-<open-catalog-account-name>.snowflakecomputing.com/polaris/api/catalog
+iceberg_rest_catalog_client_id: <open-catalog-connection-client-id>
+iceberg_rest_catalog_client_secret: <open-catalog-connection-client-secret>
+iceberg_rest_catalog_prefix: <open-catalog-name>
+
+# Optional
+iceberg_translation_interval_ms_default: 1000
+iceberg_catalog_commit_interval_ms: 1000
+----
++
+Use your own values for the following placeholders:
++
+--
+- `<snowflake-orgname>` and `<open-catalog-account-name>`: Your https://docs.snowflake.com/en/sql-reference/sql/create-catalog-integration-open-catalog#required-parameters[Open Catalog account URI] is composed of these values.
++
+TIP: In Snowflake, navigate to **Admin**, then **Accounts**. Click **...** to the right of your Open Catalog account name, and select **Manage URLs**. The **Current URL** contains `<snowflake-orgname>` and `<open-catalog-account-name>`.
+- `<open-catalog-connection-client-id>`: The client ID of the service connection you created in an earlier step.
+- `<open-catalog-connection-client-secret>`: The client secret of the service connection you created in an earlier step.
+- `<open-catalog-name>`: The name of your catalog in Open Catalog.
+--
++
+[,bash,role=no-copy]
+----
+Successfully updated configuration. New configuration version is 2.
```

Comment:
In other docs do we call out when we need to restart? Just noting that a restart is required after this point